base/doc/ltnews38.tex

```diff
@@ -148,6 +148,14 @@ \section{Sockets and plugs}
 
 For documentation see \texttt{texdoc ltsockets-doc} for now.
 
+\section{Providing \pkg{xtemplate} in the format}
+
+In \LaTeX{} News~32, we decribed the move of one long-term experimental
```

Comment:
typo: decribed > described

Reply:
Yeah - I have to actually write this yet :)